librocksdb_sys/crocksdb/c.cc

```diff
@@ -815,14 +815,12 @@ void crocksdb_multi_get(
     if (statuses[i].ok()) {
       values_list[i] = CopyString(values[i]);
       values_list_sizes[i] = values[i].size();
-      errs[i] = nullptr;
     } else {
       values_list[i] = nullptr;
       values_list_sizes[i] = 0;
       if (!statuses[i].IsNotFound()) {
-        errs[i] = strdup(statuses[i].ToString().c_str());
-      } else {
-        errs[i] = nullptr;
+        err = strdup(statuses[i].ToString().c_str());
```

Comment:
This is wrong. `err` is a pointer which will be passed by value. Use `char** err` and `SaveError` instead. And should drop all the content in values_list too.